### DISCUSSION TITLE
ServiceStack package version incompatibility

### DISCUSSION THREAD

**Discussion by mylvghb** [ServiceStack/Discuss#93] [General]
When I create a new project from a template using `x new`, the ServiceStack version defaults to **6.*** and some packages have version **8**, which prevents the project from running straight away and requires a manual upgrade of the ServiceStack package version in order to run.

![image](https://github.com/ServiceStack/Discuss/assets/35945793/0f9c6095-4f33-4e7f-b510-fa698dccb3d0)

**Comment by mylvghb** [comment]
This issues from `x mix` gists.

**Reply by mythz** [reply]
Which ServiceStack Project defaults to **6.**? Can you provide the full command you're using?

**Reply by mylvghb** [reply]
`x new web WebApp`, `x new empty Hello`, `x new grpc NorthwndApi`, ect.
![image](https://github.com/ServiceStack/Discuss/assets/35945793/03cd75dc-ecda-4a26-b792-99904159097d)

**Reply by mylvghb** [reply]
Using `x mix` auto added Nuget package version is  8.
![image](https://github.com/ServiceStack/Discuss/assets/35945793/af76988a-9e67-4200-9876-31dccfe233d0)

**Reply by mylvghb** [reply]
This problem occurs with `x new grpc NorthwindApi` and `x mix autocrudgen sqlite northwind.sqlite` commands, You can try it.

**Reply by mylvghb** [reply]
And the default .net version of the created project is still 6, while the documentation describes it as 8.
![image](https://github.com/ServiceStack/Discuss/assets/35945793/3088b8ee-bad8-45b7-a7f7-18653c860ba0)
![image](https://github.com/ServiceStack/Discuss/assets/35945793/49d30fe5-61bc-48cf-8cef-a9b7deed9f1e)

**Reply by mylvghb** [reply]
Hopefully, we can keep the version consistent

**Comment by mythz** [comment]
ok thanks we seemed to have missed the grpc template which is upgraded to `.net8.0` now. 

All the mix templates that installs packages, installs the latest version which is now **8.0.0**

**Reply by mylvghb** [reply]
Ok, I've upgraded `x` tool to the latest version and it works now!

**Comment by mylvghb** [comment]
The `blazor-server` and `blazor-wasm` project templates are still `.net6.0`.

**Reply by mythz** [reply]
We're still in the process of trying to upgrade them, however they're now deprecated.

New Blazor Templates you should use for Blazor are [blazor](https://github.com/NetCoreTemplates/blazor) or [blazor-vue](https://github.com/NetCoreTemplates/blazor-vue) templates.

See the release notes for info about the new templates:
https://docs.servicestack.net/releases/v8_00

We're still working on creating a new Web Assembly Template with .NET 8's new rendering modes, after we release that we'll be archiving and moving `blazor-wasm`, `blazor-server` and `blazor-tailwind` to Legacy Templates.
